pom.xml

```diff
@@ -96,6 +96,35 @@
       <artifactId>junit-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- required to resolve
+    These dependencies are required to workaround a bad
+    interaction using includeDependencyManagement=true for
+    the Bnd plugins and the enforcer plugin which causes a
+    false positive from the enforcer plugin. The Bnd 5.2.0
+    plugins don't have this problem, so these dependencies
+    can be removed when updating bnd.version to 5.2.0 and
+    setting them to use includeDependencyManagement=true.
```

Comment:
does this comment apply for both `includeDependencyManagement` in bnd-testing-maven-plugin and bnd-resolver-maven-plugin?

Reply:
Yes. I will improve the comment to be more specific.